webserver/templates/user/auth.html

```diff
@@ -0,0 +1,32 @@
+{% extends 'base.html' %}
+
+{% block title %}User "{{ user_id }}" - ListenBrainz{% endblock %}
+
+{% block content %}
+  <div id="user-profile">
+    <h2 class="page-title">{{ user_id }}</h2>
+
+    {% if msg %}
+    <p> {{ msg }} </p>
+
+    {% else %}
+    <h3>Approve this client for account <b>{{ user_id }}</b>?</h3>
+    <form method='POST'>
+        <input type='hidden' name='token' value='{{ token }}'/>
+        <input type='submit' value='Yes please'/>
+    </form>
+    {% endif %}
+
+  </div>
+{% endblock %}
+
+{% block scripts %}
+  {{ super() }}
```

Comment:
What's this? It looks like it's from the user profile page. Make sure that when you copy files for a new task that you remove all unneeded code.